lib/PatternMultiTopicsConsumerImpl.cc

```diff
@@ -118,6 +118,9 @@ void PatternMultiTopicsConsumerImpl::timerGetTopicsOfNamespace(const Result resu
     // callback method when added topics all subscribed.
     ResultCallback topicsAddedCallback = [this, topicsRemoved, topicsRemovedCallback](Result result) {
         if (result == ResultOk) {
+            if(messageListener_) {
```

Comment:
```suggestion
            if (messageListener_) {
```

Fix the coding style